<!-- -->
<launch>
    <!--points_process-->
    <node pkg="points_process" type="points_process_node" name="points_process_node" output="screen">
        <param name="sub_topic" value="/converted_velodyne_points"/>
        <param name="pub_topic" value="/lidar_points_processed"/>
        <param name="crop_view_mode" value="true"/>
        <param name="crop_range_mode" value="true"/>
        <param name="downsample_mode" value="true"/>
        <param name="filter_mode" value="false"/>
        <param name="show_points_size" value="true"/>
        <param name="show_time" value="true"/>
        <param name="view_number" value="2"/>
        <param name="field_of_view" value="130"/>
        <param name="sensor_height" value="4.0"/>
        <param name="view_higher_limit" value="100.0"/>
        <param name="view_lower_limit" value="-100.0"/>
        <param name="min_distance" value="1.5"/>
        <param name="max_distance" value="50.0"/>
        <param name="leafsize" value="0.15"/>
        <param name="meank" value="5"/>
        <param name="stdmul" value="0.2"/>
    </node>
    
    <!--points_ground_filter-->
    <node pkg="points_ground_filter" type="points_ground_filter_node" name="points_ground_filter_node" output="screen">
        <param name="sub_topic" value="/lidar_points_processed"/>
        <param name="pub_ground_topic" value="/lidar_points_ground"/>
        <param name="pub_no_ground_topic" value="/lidar_points_no_ground"/>
        <param name="show_points_size" value="true"/>
        <param name="show_time" value="true"/>
        <param name="crop_range_mode" value="true"/>
        <param name="range_front" value="50"/>
        <param name="range_rear" value="50"/>
        <param name="range_left" value="50"/>
        <param name="range_right" value="50"/>
        <param name="sensor_height" value="4.0"/>
        <param name="seeds_distance_threshold" value="1.0"/>
        <param name="ground_distance_threshold" value="0.2"/>
        <param name="num_lpr" value="100"/>
        <param name="seg_num_front" value="3"/>
        <param name="seg_num_rear" value="3"/>
        <rosparam param="seg_distance_front" > [20, 40, 60] </rosparam>
        <rosparam param="seg_distance_rear" > [20, 40, 60] </rosparam>
    </node>
</launch>
